### PR TITLE
movie: Virus (1999) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Setting the film in the eye of a typhoon was a neat idea, but that's where any novelty stopped. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 1]
```diff
@@ -11501,6 +11501,42 @@ biological weapons [The Drakh contaminated Earth with biogenic plague Earth usin
...
+incorporeal being [A malevolent electrical energy being from outer space was inhabiting the machinery of the Russian science vessel Volkov.]
+speculative robot [The malevolent electrical energy being was going around killing people in a giant robot body that it was controlling.]
+threat from extraterrestrial life [The film culminated with Foster and Nadia realizing that malevolent electrical entity sought to seize a British intelligence station from which it could seize control of the world's military forces and use them exterminate humanity. They sent the life form to the bottom of the sea rather than let it achieve its horrible end.]
+
+:: Minor Themes
+human vs. nature [The crew of the tugboat Sea Star nearly lost their lives while sailing through a typhoon.]
+alcohol abuse [The tugboat Sea Star was captained by the alcoholic old man Robert Everton.]
+
+:: Not Themes
+space station [A malevolent electrical entity attacked the Russian space station Mir. But this theme does not apply since Mir was an actually existing space station that was in orbit around Earth at the time.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1990s.st.txt:11537]: I think we shouldn't split hairs about such cutting edge science things as space stations. If the story is set in the future it is probably reasonable to say that it portrayed a speculative future version of that Russian **space station**. At any rate I wager the author is **speculating** a lot about it. Naturally I wouldn't want to extend that logic to, say, apples which are everyday objects today, and the future of which authors are rather unlikely to speculate about.

**Review by odinlake — COMMENTED** [review]
Comment inline...

File: notes/stories/film/film-scifi-1990s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt:11537]: Points of order:
- Set in the present day.
- Showed the actually existing space station Mir.

No objections to using this theme to catch actual space stations, as they are kind of on the cutting edge of what is currently possible. But we can't justify it by claiming the viewer is shown a future version of Mir or anything like that.

Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt]: Okay, moved **space station** to minor and pushed changes.

**Review by odinlake — APPROVED** [review]

**Pull request merged**